src/monda/sorad/plots.py

```diff
@@ -151,79 +152,160 @@ def plot_rrs_qc_3c(rrs, time, wl, q_1, q_2, q_3, file_id, target):
 
         plt.subplot(2, 2, 1)
-        plt.title(f"No QC (n = {int(len(q_1))})")
+        plt.title(f"No QC (n = {int(len(q_0))})")
         plt.plot(wl, rrs.T, linewidth=0.4, alpha=0.6)
         plt.xlim(350, 950)
         plt.ylim(-0.002, ymax) # force axis limits
-        #plt.grid()
+        plt.grid()
         plt.xlabel('Wavelength [nm]')
         plt.ylabel('$R_{rs}$ [sr$^{-1}$]')
 
         plt.subplot(2, 2, 2)
-        plt.title(f"Rad QC (n = {int(np.sum(q_1))})")
-        plt.plot(wl, rrs.T[:,q_1==1], linewidth=0.4, alpha=0.8)
+        plt.title(f"$q_{0}$ (n = {int(np.sum(q_0))})")
+        plt.plot(wl, rrs.T[:,q_0==1], linewidth=0.4, alpha=0.8)
         plt.xlim(350, 950)
         plt.ylim(-0.002, ymax) # force axis limits
-        #plt.grid()
+        plt.grid()
         plt.xlabel('Wavelength [nm]')
 
         plt.subplot(2, 2, 3)
-        plt.title(f"Rad + Rrs QC (n = {int(np.sum(q_2))})")
-        plt.plot(wl, rrs.T[:,q_2==1], linewidth=0.4, alpha=0.8)
+        plt.title(f"$q_{1}$ (n = {int(np.sum(q_1))})")
+        plt.plot(wl,rrs.T[:,q_1==1], linewidth=0.4, alpha=0.8)
         plt.xlim(350, 950)
         plt.ylim(-0.002, ymax) # force axis limits
+        plt.grid()
+        plt.xlabel('Wavelength [nm]')
+
+       # plt.subplot(2, 3, 4)
+        #plt.title(f"$q_{2}$ (n = {int(np.sum(q_2))})")
+        #plt.plot(wl,rrs.T[:,q_2==1], linewidth=0.4, alpha=0.8)
+        #plt.xlim(350, 950)
+        #plt.ylim(-0.002, ymax) # force axis limits
         #plt.grid()
+        #plt.xlabel('Wavelength [nm]')
+        
+        plt.subplot(2, 2, 4)
+        plt.title(f"$q_{3}$ (n = {int(np.sum(q_3))})")
+        plt.plot(wl,rrs.T[:,q_3==1], linewidth=0.4, alpha=0.8)
+        plt.xlim(350, 950)
+        plt.ylim(-0.002, ymax) # force axis limits
+        plt.grid()
         plt.xlabel('Wavelength [nm]')
+        plt.ylabel('$R_{rs}$   [sr$^{-1}$]')
 
+
+        plt.tight_layout()
         plt.subplots_adjust(hspace=0.5)
 
         plt.savefig(os.path.join(target, file_id + '_Rrs_QC.png'), format='png', dpi=150)
 
     return
 
 
-def plot_coveragemap(lat, lon , q, file_id, target, map_resolution=11):
+
+
+def plot_coveragemap(lat, lon , q, file_id, target, map_resolution=10):
     """ coverage map showing quality control filtered data: color scheme matches
     `results' plot"""
     if np.sum(q) > 0:
         colors= cm.cool(np.linspace(0, 1, int(sum(q))))
 
         plt.figure(figsize=(15,10))
-        extent = [np.floor(np.min(lon*100))/100, np.ceil(np.max(lon*100))/100, np.floor(np.min(lat*100))/100, np.ceil(np.max(lat*100))/100]
-        request = cimgt.StamenTerrain()
+        extent = [np.floor(np.min(lon*10))/10, np.ceil(np.max(lon*10))/10, np.floor(np.min(lat*10))/10, np.ceil(np.max(lat*10))/10]
+        request =  cimgt.GoogleTiles(style='satellite')
         ax = plt.axes(projection=ccrs.PlateCarree())
-        ax.add_image(request, map_resolution)
+        ax.add_image(request, 10)
```

Comment:
Hi Stefan - just to make sure I am doing this right; should I make changes/new commits to my local version of the repo, then send a new pull request when done?

For this function, I think it can be useful to have the map resolution (e.g. for some underway data, over a larger area).

I will removed the hardcoding